services/frontend/src/components/ScrollContainer/ScrollContainer.js

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import ScrollContainerLayout from "./ScrollContainerLayout";
 
-const ScrollContainer = ({children}) => (
-    <ScrollContainerLayout>
+const ScrollContainer = ({children, className}) => (
+    <ScrollContainerLayout className={className+ "-page"}>
```

Comment:
As we discussed, instead of having an implicit `-page` postfix for classnames, we should instead make use of the `classNames` library to combine a default classname with a custom classname when instantiating the component.